### DISCUSSION TITLE
Reader suddenly broken

### DISCUSSION THREAD

**Discussion by VMedic77** [sumatrapdfreader/sumatrapdf#2499] [Q&A]
Hi
I was using Sumatra for a while and it was small,fast and clean solution. After switching from win 7 to win 10, after a while,my reader doesnt show top 1/7-th of a screen (approx)...it is just blank white or black. When I press shortcuts, it all works, but I just cant see it. I tried 3.3.3 and pre-realese 3.4, same thing. I have no clue what is going on,but noticed that when I click on that blank area, whichever desktop icon that is on that position where I click (that I cant see) activates,like that part of a screen is messed up completely and out of the application. Any thoughts ?

![Screenshot 04-23-2022 11 46 40](https://user-images.githubusercontent.com/104254368/164889435-3664b1ed-78d2-4868-b1e0-ebb3ff630899.png)

**Comment by GitHubRulesOK** [comment]
never seen anything like that on various machines so try to get a screenshot of differing views perhaps with sidebar
![image](https://user-images.githubusercontent.com/12754913/164892175-0f64717a-53f1-41b6-b398-5653a9125299.png)

if the portable version https://www.sumatrapdfreader.org/dl/prerel/14472/SumatraPDF-prerel.exe works its perhaps a settings issue but if portable does not function it is perhaps a graphics driver problem
![image](https://user-images.githubusercontent.com/12754913/164892335-99aa3098-ba2f-49c1-9a20-f8cba1d8f046.png)

**Comment by VMedic77** [comment]
Thank u for reply.
 Now that u mentioned it, I investigated further and realized it is indeed graphic driver problem. I have multiple monitors and SCALE FULL SCREEN on secondary monitor must be checked and override app settings must be checked to  fix this issue.  (IntelHD graphics)
Greetings

**Reply by GitHubRulesOK** [reply]
there are many ways to change an apps graphics but perhaps check if you need to alter some of those properties in driver or application
![image](https://user-images.githubusercontent.com/12754913/164892794-da5a5b4a-c77d-49d0-a056-dad494141d58.png)
